assets/js/base/context/cart-checkout/checkout-state/index.js

```diff
@@ -305,7 +305,8 @@ export const CheckoutStateProvider = ( {
 		isSuccessResponse,
 	] );
 
-	const onSubmit = useCallback( () => {
+	const onSubmit = useCallback( ( event ) => {
+		event.preventDefault();
```

Comment:
Are we calling `preventDefault` twice? Here and inside `CheckoutForm` `onFormSubmit`.

Reply:
Ooops this was from an early iteration. I forgot to remove this, good catch!